projects/gnomad/src/DownloadsPage.js

```diff
@@ -1,11 +1,334 @@
+  ],
+  [
+    '20.18 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr9.vcf.bgz',
+  ],
+  [
+    '21.38 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr10.vcf.bgz',
+  ],
+  [
+    '21.99 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr11.vcf.bgz',
+  ],
+  [
+    '21.42 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr12.vcf.bgz',
+  ],
+  [
+    '15.04 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr13.vcf.bgz',
+  ],
+  [
+    '14.74 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr14.vcf.bgz',
+  ],
+  [
+    '13.87 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr15.vcf.bgz',
+  ],
+  [
+    '15.54 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr16.vcf.bgz',
+  ],
+  [
+    '13.62 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr17.vcf.bgz',
+  ],
+  [
+    '12.16 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr18.vcf.bgz',
+  ],
+  [
+    '11.3 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr19.vcf.bgz',
+  ],
+  [
+    '9.69 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr20.vcf.bgz',
+  ],
+  [
+    '6.17 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr21.vcf.bgz',
+  ],
+  [
+    '6.54 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chr22.vcf.bgz',
+  ],
+  [
+    '17.92 GiB',
+    'https://storage.googleapis.com/gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.chrX.vcf.bgz',
+  ],
+]
+
+const createVcfLink = file => {
+  const fileSize = file[0]
+  const url = file[1]
+  const chrom = /chr(\w+)/.exec(url)[0]
+  const tabixUrl = `${url}.tbi`
+  console.log(chrom)
+  return (
+    <li>
+      <ExternalLink href={url}>{`${chrom} sites VCF (${fileSize}) `}</ExternalLink>
+      <ExternalLink href={tabixUrl}>{`(.tbi)`}</ExternalLink>
+    </li>
+  )
+}
+
 export default () => (
   <InfoPage>
     <PageHeading>Downloads</PageHeading>
+    <p>
+      The variant dataset files below contain all subsets and are large in size. We recommend using{' '}
+      <ExternalLink href={'https://hail.is/'}>Hail</ExternalLink> to work with gnomAD data. It is
+      easiest download these files in parallel on the commandline using{' '}
```

Comment:
* "easiest **to** download"
* "command line" is two words or hyphenated
* gsutil vs gutils